Sources/SourceKitLSP/Swift/SwiftLanguageService.swift

```diff
@@ -432,7 +437,7 @@ extension SwiftLanguageService {
     cancelInFlightPublishDiagnosticsTask(for: notification.textDocument.uri)
     await diagnosticReportManager.removeItemsFromCache(with: notification.textDocument.uri)
 
-    guard let snapshot = self.documentManager.open(notification) else {
+    guard let snapshot = try? self.documentManager.open(notification) else {
```

Comment:
Could you use `orLog` here as well?

```suggestion
    guard let snapshot = orLog("Getting document manager", { try self.documentManager.open(notification) }) else {
```